servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1783,65 +1795,72 @@ private void setStateFromEntry(SetStateEntry entry) throws FileDoesNotExistExcep
   }
 
   /**
-   * Checks user's permission on a path.
+   * Checks user's permission on a path. If the path is invalid, it should bypass the
+   * {@link InvalidPathException} and the logic at operation will handle it.
```

Comment:
Could you add a comment that the caller should lock `mInodeTree` before calling it?